add-item.html

```diff
@@ -2,15 +2,228 @@
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class="checkbox">
+                            <h3>Choose everything that applies.</h3>
+
+                            <div class="checkbox__options">
+                                <div class="checkbox__option">
+                                    <input type="checkbox" name="way" id="checkbox__option" value="student" />
+                                    <label for="checkbox__option">Student</label>
+                                </div>
+
+                                <div class="checkbox__option">
+                                    <input type="checkbox" name="way" id="checkbox__option" value="teacher" />
+                                    <label for="checkbox__option">Teacher</label>
+                                </div>
+
+                                <div class="checkbox__option">
+                                    <input type="checkbox" name="way" id="checkbox__option" value="businessman" />
+                                    <label for="checkbox__option">Business owner</label>
+                                </div>
+
+                                <div class="checkbox__option">
+                                    <input type="checkbox" name="way" id="checkbox__option" value="freelancer" />
+                                    <label for="checkbox__option">Freelancer</label>
+                                </div>
+
+                                <div class="checkbox__option">
+                                    <input type="checkbox" name="way" id="checkbox__option" value="traveller" />
+                                    <label for="checkbox__option">Traveller</label>
+                                </div>
+
+                                <div class="checkbox__option">
+                                    <input type="checkbox" name="way" id="checkbox__option" value="tourist" />
+                                    <label for="checkbox__option">Tourist</label>
+                                </div>
+
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="select__date">
+
+                        <h3>When do you plan to travel?</h3>
+                        <div class="date">
+                            <input type="date" name="startDate" id="travel_date" required />
+                            <input type="date" name="endDate" id="travel_date" required />
+                        </div>
+
+                    </div>
+                </div>
+                <div class="button">
+                    <button type="submit" class="btn">Submit</button>
+                </div>
+        </div>
+        </form>
+    </div>
+    </div>
+
+    <div id="modal">
+        <div id="modal-content">
+            <p>Form submitted successfully!</p>
```

Comment:
forgot to add other requirements !!!
Accept and decline actions

Reply:
done!